modules/ROOT/pages/batch-filters-and-batch-aggregator.adoc

```diff
@@ -392,6 +392,19 @@ The previous error occurs because MIME types are not preserved by default, and t
 
 By setting this attribute, Mule automatically maintains each record's media type and knows that the payload actually represents a JSON document.
 
+== Max Concurrency Limit
+
+You can limit the number of records that a batch job can process by setting the `maxConcurrency` parameter as in the following example:
```

Comment:
Maybe adding some further explanation like "In a strict sense, maxconcurrency limit the number of blocks that are concurrently processed. The records in each block are processed sequentially. As a consequence of this, the maximum number of records that are concurrently processed corresponds to the maxConcurrency param"